src/router/constants/crumbs.ts

```diff
@@ -63,6 +63,11 @@ export const myCourses = {
   path: authRoutes.myCourses.root.route
 }
 
+export const newCourse = {
+  name: t('breadCrumbs.myCourses'),
```

Comment:
should be newCourse

Reply:
newCourse breadcrumb was returned in new commit